tests/acceptance/helpers/sharingHelper.js

```diff
@@ -167,5 +188,25 @@ module.exports = {
           })
       }
     }
+  },
+  assertUserLastPublicShareDetails: async function (linkCreator, expectedDetails) {
+    const lastShare = await this.fetchLastPublicLinkShare(linkCreator)
+    if (lastShare.share_type === 3) {
+      const regDate = lastShare.expiration.split(/ /)
+      if (
+        Object.prototype.hasOwnProperty.call(expectedDetails, 'token') &&
```

Comment:
Any reason to check for `hasOwnProperty`?

Reply:
What do you think? Is it not necessary??

Reply:
I guess not.

Reply:
Ok then..